kubetest2-gke/deployer/network.go

```diff
@@ -0,0 +1,331 @@
+/*
```

Comment:
Thanks for moving it to a separate file.
I'm not too aware of the functionality being added here, but mostly looks like it's following: https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-shared-vpc#creating_a_network_and_two_subnets
whigh LGTM

high level nit: let's break down most of these into helper functions (apart from createnetwork and setupnetwork) by removing the deployer receiver and pass whatever is required (projects/network names) as arguments.

Reply:
Yes, it's mostly following that doc.
I have updated most of them into helper functions, as you suggested.